e2e/test/project.test.ts

```diff
@@ -20,6 +52,11 @@ async function createProject(page: Page, project: UpsertProject) {
   await expect(page).toHaveURL(/https:\/\/localhost:1443\/hanke\/[0-9a-f-]+/);
   const projectId = page.url().split('/').at(-1);
 
+  // NOTE: Geometry edit request with API call (not using the UI)
+  await page.request.post('https://localhost:1443/trpc/project.updateGeometry', {
```

Comment:
[nit] Could the tRPC client be used here? Then there would be no need for SuperJSON wrapping and typings would also work.